### DISCUSSION TITLE
Test merge request vulnerabilities

### DISCUSSION THREAD

**Discussion by alexr3** [DefectDojo/django-DefectDojo#9586] [Q&A] [answered]
Hello everyone!

How can make, that test MR vulnerabilities that will not go into the prod branch, are in the Dojo separately from the production vulnerabilities?

That is, the vulnerabilities of test from the vulnerabilities of production?

**Comment by manuel-sommer** [comment, marked as answer]
Hi @alexr3,

what you could do is make two CI/CD engagements and use Branch / Tag:

![grafik](https://github.com/DefectDojo/django-DefectDojo/assets/47991713/7f1a3aa7-6dfe-4d18-b833-e9b8bcf4d5db)

Furthermore, you could activate that deduplication is only done within the engagment:

![grafik](https://github.com/DefectDojo/django-DefectDojo/assets/47991713/32ff37ae-8170-4f1b-a9cf-0de9bfed63b5)

Then, you don't deduplicate findings within one product which come from various environments.

Best Regards,

Manuel

**Comment by manuel-sommer** [comment]


**Reply by mtesauro** [reply]
GH isn't giving me the option to do that on this one for some reason.

**Reply by manuel-sommer** [reply]
@mtesauro, could you change the category of idea to "Question" and then you can mark this as "answered"?
